components/director/internal/domain/label/translator.go

```diff
@@ -0,0 +1,15 @@
+package label
+
+import "regexp"
+
+// ExtractValueFromJSONPath returns the value that is placed in the SQL/JSON path query
+func ExtractValueFromJSONPath(jpq string) *string {
```

Comment:
IMO in this method, we should be strict and if a query does not match regexp, return an error